index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <header id="header">
+  <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSt_S0TvjguVUdnKEheN1MgkcuIVWAlTOVWoA&usqp=CAU" alt="" id="header-img">
```

Comment:
so far you are showing a great effort but you need to make some changes. 
it would be better if you put the img in a container like div. it will become easy for styling.